### ISSUE TITLE
Add support for custom error messages

### ISSUE THREAD

**Issue by petebrowne** [localmed/pyserializer#12]
The fields should support custom error messages, i.e.:

``` python
class Person(Serializer):
  age = IntegerField(error_messages={'invalid': 'oops'})
```

**Comment by joeljames** [comment]
@petebrowne If you haven't started working on this issue and is still interested in this feature, I can start work on this a submit a PR. Just wanted to check with you before I do any work on this.

**Comment by petebrowne** [comment]
Hey @joeljames! We haven't started working on this, so feel free

**Comment by joeljames** [comment]
The issue has been addressed in PR https://github.com/localmed/pyserializer/pull/23.
The feature is available in pyserializer 0.8.0 https://github.com/localmed/pyserializer/releases/tag/0.8.0
